### PR TITLE
Complete PHPStan Level 6

### LINKED ISSUE #4112: PHPStan level 6
PHPStan is an automated scanner, which reports code that could be improved https://phpstan.org

We already pass PHPStan level 5 since https://github.com/FreshRSS/FreshRSS/pull/4110

Usage:

```sh
# See https://getcomposer.org
composer install --prefer-dist
composer run-script phpstan
```

Now it would be nice to progress to PHPStan level 6 *report missing typehints* https://phpstan.org/user-guide/rule-levels

https://github.com/FreshRSS/FreshRSS/blob/1335a0e3cf11a0d4248e9eaaf748b89e6df741ef/phpstan.neon#L2-L3

```sh
# Manual override of the analyse level
vendor/bin/phpstan analyse . --level 6
```

Help welcome, and it is fine to send multiple PRs, each solving some of the PHPStan warnings (best if covering at least one or more files fully).

When you make more files pass Level 6, you can remove them ([example](https://github.com/FreshRSS/FreshRSS/commit/a5cc7b20f6b05ceae73a90dc418345f7f0dede6f)) from our [list of non-compliant files](https://github.com/FreshRSS/FreshRSS/blob/edge/tests/phpstan-next.txt) (cf. https://github.com/FreshRSS/FreshRSS/pull/5230 )

```sh
composer run-script phpstan-next
```

### PR DESCRIPTION
**Pull request by Alkarex**
Fix https://github.com/FreshRSS/FreshRSS/issues/4112
And initiate PHPStan Level 7

### CODE REVIEW

**Comment by Alkarex** [comment]
* [x] ~~We still lack the `./tests/` folder though, if you could be interested @ColonelMoutarde~~

https://github.com/FreshRSS/FreshRSS/blob/efcc8f387bb44901e0e3e284b7aa225e684549fd/phpstan.neon#L17-L18

Edit: Done!

File: lib/Minz/ConfigurationSetterInterface.php
Comment (Alkarex) [inline lib/Minz/ConfigurationSetterInterface.php:1]: To replace unused class removed in https://github.com/FreshRSS/FreshRSS/pull/5251

**Review by ColonelMoutarde — APPROVED** [review]

**Comment by math-GH** [comment]
Awesome final run. Great work!

File: app/Models/Feed.php [hunk 1]
```diff
@@ -708,7 +712,7 @@ public function loadHtmlXpath() {
 				if ($item['title'] != '' || $item['content'] != '' || $item['link'] != '') {
 					// HTML-encoding/escaping of the relevant fields (all except 'content')
 					foreach (['author', 'categories', 'guid', 'link', 'thumbnail', 'timestamp', 'title'] as $key) {
-						if (!empty($item[$key])) {
+						if (!empty($item[$key]) && is_string($item[$key])) {
```
Comment (Alkarex) [inline app/Models/Feed.php:715]: Regression https://github.com/FreshRSS/FreshRSS/discussions/7325

Comment (Alkarex) [reply app/Models/Feed.php:715]: Fixed in https://github.com/FreshRSS/FreshRSS/pull/7345

**Pull request merged**